packages/dds/test-dds-utils/src/ddsFuzzHarness.ts

```diff
@@ -507,6 +521,7 @@ export const defaultDDSFuzzSuiteOptions: DDSFuzzSuiteOptions = {
 	detachedStartOptions: {
 		numOpsBeforeAttach: 5,
 	},
+	handles: false,
```

Comment:
we should improve naming here a bit. i would also probably invert it, as the end state is on by default, and only some tests might disable it. so something like, handleGenerationDisable?: boolean